roles/horizon/tasks/logging.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: set up log rotation for horizon
+  logrotate: name=horizon path=/var/log/apache2/*.log
+  args:
+    options:
+      - daily
+      - missingok
+      - rotate 7
+      - compress
```

Comment:
Our logrotate module appears to support having scripts and other options. I would suggest attempting to get it all done through the module, and if that isn't functional, filing a defect against our module in Ursula, and falling back to your template suggestion.